Sources/SafeDICore/Visitors/InstantiableVisitor.swift

```diff
@@ -138,16 +138,10 @@ public final class InstantiableVisitor: SyntaxVisitor {
     }
 
     public override func visit(_ node: ExtensionDeclSyntax) -> SyntaxVisitorContinueKind {
-        guard declarationType.isExtension else {
-            return .skipChildren
-        }
-        guard let instantiableMacro = node.attributes.instantiableMacro else {
-            // Not an instantiable type. We do not care.
-            return .skipChildren
-        }
```

Comment:
again, this was super dead code – no way to reach it given how it's called. we were protecting against nothing.